src/renderer/services/api/http/httpClient.ts

```diff
@@ -1,13 +1,16 @@
 import axios, {AxiosError, AxiosInstance, AxiosRequestConfig, AxiosResponse} from "axios";
+import database from "../../db/api/database";
 
 export class HttpClient {
     private client: AxiosInstance;
+    private readonly loggerKey?: string;
 
-    public constructor(baseURL: string, options: {axios?: AxiosRequestConfig} = {}) {
+    public constructor(baseURL: string, options: {axios?: AxiosRequestConfig; loggerKey?: string} = {}) {
         if (!options) {
             // eslint-disable-next-line no-param-reassign
             options = {axios: {}};
         }
+        this.loggerKey = options.loggerKey;
```

Comment:
I think loggerKey should be baseURL?

Reply:
it could be but is not exclusive to be baseURL
like in case if you want record metrics for every validator, but these validators use the same beacon node, whit baseURl you will be unable to separate results

Reply:
hmm, but validator metrics are not collected here? We collect beacon node metrics here which is uniquely identified by url

Reply:
HTTP client is not strictly linked to beacon, can be used in validator too or any other HTTP requests

a possible problem with the baseURL, you can have validator on `localhost:5052` so the baseURL will be `http://localhost:5052/eth/v1/` or even `http://localhost:5052/eth/v1/beacon` or there is a new version of APIs so old logs will be "forgotten"